numpy/lib/function_base.py

```diff
@@ -2266,17 +2266,17 @@ class vectorize:
 
     Decorator syntax is supported.  The decorator can be called as
     a function to provide keyword arguments.
-    >>>@np.vectorize
-    ...def identity(x):
-    ...    return x
+    >>> @np.vectorize
```

Comment:
```suggestion

    >>> @np.vectorize
```
(Actually, I think a blank line before is also nicer, although probably doesn't matte rhere.).